### PR TITLE
Please Review My Mobile Portfolio Skeleton Version

### PR DESCRIPTION
**Pull request by efoosag**
This is the first request for this project. thank you for the review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  </head>
+  <body>
+    <section class="wrapper">
+      <section class="header">
+        <div class="logo"><img src="./images/my-logo.svg" alt="My Logo" /></div>
+        <div class="icon"></div>
+      </section>
+      <section>
+        <section class="main">
+          <h2>
```
Comment (Lordkaito) [inline index.html:18]: - [ ] Please change this `h2` tag into an `h1` tag, as this is the main title of your page, you should be as semantic as possible.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @efoosag, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] [OPTIONAL] This is not one mandatory requirement for you to pass this milestone, but it would be great if you can fix the horizontal scroll on your app. Right now, it does not really matter how big or small your screen is that you are going to have some horizontal scrolling. This may be caused by too big sizes used during your styling or because there's too much content inside any container. I would really recommend you to take a look and fix this issue.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>Portfolio Setup And Mobile Version Skeleton</title>
+  </head>
+  <body>
+    <section class="wrapper">
+      <section class="header">
```
[the remark this answers is not in the document]
Comment (efoosag) [reply index.html]: I have effected the Changes

File: index.html (hunk 1 above)
Comment (efoosag) [reply index.html]: Resolved

File: index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (Whoistolu) [reply index.html]: 👍🏽 All good now.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <div class="logo"><img src="./images/my-logo.svg" alt="My Logo" /></div>
+        <div class="icon"></div>
+      </section>
+      <section>
```
[the remark this answers is not in the document]
Comment (Whoistolu) [reply index.html]: 👍🏽 All good now.

File: index.html (hunk 1 above)
Comment (Whoistolu) [reply index.html]: 👍🏽 All good now.

**Review by Whoistolu — APPROVED** [review]
### Hi @efoosag,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**